dspace-xmlui/src/main/java/org/dspace/app/xmlui/objectmanager/ItemAdapter.java

```diff
@@ -761,20 +764,41 @@ protected void renderFileSection(Context context) throws SQLException, SAXExcept
                 continue;
             }
 
+            // /////////////////////////////////////
+            // Determine which bitstreams to include in bundle
+            List<Bitstream> bitstreams = new ArrayList<Bitstream>();
+
+            // If this is the THUMBNAIL bundle, and we are NOT including content bundle,
+            // Then assume this is an item summary page, and we can just include the main thumbnail.
+            if (isDerivedBundle && !includeContentBundle)
+            {
```

Comment:
The derived bundle could be TEXT or THUMBNAIL.  Should this check explicitly for the thumbnail bundle?

Reply:
+1 . mind you if you look a few lines down, if it's TEXT it'll return null on the ItemService.getThumbnail() call and so the bitstream won't be added to the bitstreams list anyway

Reply:
oh, no, i'm wrong about that.. it might mean the thumbnail is added twice if the item has both THUMBNAIL and TEXT since the 'getThumbnail' is really just against the item and not the bundle..